keylime/tpm/tpm_main.py

```diff
@@ -1157,11 +1216,22 @@ def check_quote(
         if hash_alg is None:
             hash_alg = self.defaults["hash"]
 
+        msg, success, clock_info = self.check_quote_timing(clock_info, quote)
```

Comment:
The time checks needs to be after the quote validation because `tpm2_print` does not validate the data structure.
Can you add also a comment that states that, so the order does not get changed accidentally?

Reply:
Very good point, I will change accordingly